packages/roosterjs-editor-dom/lib/utils/adaptFontColorToBackgroundColor.ts

```diff
@@ -0,0 +1,76 @@
+const TRANSPARENT = 'transparent';
+const DARK = 'DARK';
+const BRIGHT = 'BRIGHT';
+const DEFAULT = 'DEFAULT';
+
+/**
+ * Change the font color to white or some other color, so the text can be visible with a darker background
+ * @param element The element that contains text.
+ * @param newColor The font color to be applied. If not defined, the font color will be white.
+ */
+export default function adaptFontColorToBackgroundColor(
+    element: HTMLElement,
+    newFontColorInDark?: string,
+    newFontColorInBright?: string
```

Comment:
I didn't see when these parameters are used

Reply:
I put this parameters in case the user whats to change the font color to other color rather than black/white